### PR TITLE
Refactor source, destination, memo query

### PR DESCRIPTION
**Pull request by orkunkl**

### CODE REVIEW

File: pkg/store/store.go [hunk 1]
```diff
@@ -424,36 +425,23 @@ func (s *Store) LoadTxsInBlock(ctx context.Context, blockHeight int64) ([]models
...
-			WHERE message ->> 'path' = 'cash/send' 
-			AND message -> 'details' ->> 'source' = $1
-			AND message -> 'details' ->> 'destination' = $2
-		`, source, dest)
+func (s *Store) LoadTxsByParams(ctx context.Context, source, dest, memo string) ([]models.Transaction, error) {
+	if source == "" && dest == "" && memo == "" {
+		return nil, errors.Wrap(errors.ErrInput, "no input provided")
 	}
+
+	query := "SELECT transaction_hash, block_id, message FROM transactions"
```
Comment (orkunkl) [inline pkg/store/store.go]: I need to build a dynamic(chained) query that uses 3 optional values. How to achieve this using "database/sql" @husio ?

Comment (husio) [reply pkg/store/store.go]: I don't understand what you want to achieve. Can you explain more?

Comment (orkunkl) [reply pkg/store/store.go]: SQL injection attacks came to my mind. I was trying stuff around

Comment (husio) [reply pkg/store/store.go]: What I wrote is incorrect. I will update the query soon.

**Pull request merged**